src/brevitas/graph/gptq.py

```diff
@@ -23,85 +23,6 @@
 import brevitas.nn as qnn
 
 
-class gptq_mode(gpxq_mode):
```

Comment:
Why has this been moved?

Reply:
Dependency issue as `GPTQ` needs to be defined first to be used as the default for `gptq_class` in `gptq_mode`